### PR TITLE
Tweaks for appearance of the standalone page

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/18419
*Description of changes:*

- Tailwind style tweaks to adjust the font color, height, and width within the register standalone page.
- Minor update for Tax Number validation function (e.g., "000000000 cannot pass the check; "8" cannot be the first digit)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-56-ntwfsixg.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-56-ntwfsixg.web.app

**Review by hfekete — APPROVED** [review]
Looks good, you mentioned you will add the placeholders ?
I can merge after that.

File: btr-web/btr-common-components/app.config.ts [hunk 1]
```diff
@@ -3,18 +3,18 @@ export default defineAppConfig({
     primary: 'bcGovBlue',
     gray: 'bcGovGray',
     formGroup: {
-      label: { base: 'block text-base font-bold text-gray-700 dark:text-gray-200' }
+      label: { base: 'block text-base font-bold py-3 text-black-700 dark:text-black-200' }
```
[the remark this answers is not in the document]
Comment (patrickpeinanw) [reply btr-web/btr-common-components/app.config.ts:6]: > Did changing to black changed anything ?

Yes. It makes all labels in UFormGroup black. 
Maybe we can remove the "dark:text-black-200"... do we want to have a dark mode for this web application?

File: btr-web/btr-common-components/components/bcros/inputs/CountriesOfCitizenship/Index.vue [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
+        v-model="citizenshipType"
+        :value="option.value"
+      />
+      <label :for="`radio-${option.value}`" class="ml-5">
+        <div class="text-base text-black">
+          {{ option.label }}
+        </div>
+      </label>
+    </div>
+    <div class="ml-9">
```
[the remark this answers is not in the document]
Comment (patrickpeinanw) [reply btr-web/btr-common-components/components/bcros/inputs/CountriesOfCitizenship/Index.vue:15]: > This is cool, actually making it outside of the radio buttons :)

Yeah. That will make it easier to align the third radio button and its label text.

File: btr-web/btr-common-components/app.config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-common-components/app.config.ts:6]: No, we dont need dark mode, that can be removed. 
At least for now we dont need dark mode.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-56-ntwfsixg.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-56-ntwfsixg.web.app

**Comment by patrickpeinanw** [comment]
> Looks good, you mentioned you will add the placeholders ? I can merge after that.

The placeholders have been added. Can you help me merge the PR? Thanks

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-56-ntwfsixg.web.app

File: btr-web/btr-common-components/app.config.ts [hunk 3]
```diff
@@ -3,18 +3,18 @@ export default defineAppConfig({
     primary: 'bcGovBlue',
     gray: 'bcGovGray',
     formGroup: {
-      label: { base: 'block text-base font-bold text-gray-700 dark:text-gray-200' }
+      label: { base: 'block text-base font-bold py-3 text-black-700' }
```
Comment (kialj876) [inline btr-web/btr-common-components/app.config.ts:6]: the text colour should pretty much always default to text-bcGovGray-700 or 900. I think for these text-bcGovGray-700 is probably in line with the other ones

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-56-ntwfsixg.web.app

**Review by kialj876 — APPROVED** [review]

**Pull request merged**